dimod/quadratic/quadratic_model.py

```diff
@@ -672,6 +672,10 @@ def change_vartype(self, vartype: VartypeLike, v: Variable) -> "QuadraticModel":
         self.data.change_vartype(vartype, v)
         return self
 
+    def clear(self) -> None:
+        """Remove the offset and all variables and interactions from the model."""
+        self.data.clear()
```

Comment:
...and this line calls clear in `cyqm_template.pyx.pxi` (via the corresponding data class), right?